setup.cfg

```diff
@@ -57,4 +57,5 @@ ignore-docstrings = yes
 ignore-imports = yes
 max-args = 9
 good-names = df
+disable = unspecified-encoding
```

Comment:
what issues where raised before adding this line?

Reply:
There was a pylint encoding error (unspecified-encoding (W1514)) being triggered in `sigpro/configuration.py`. It was causing the lint job to fail, and since the suggested solution isn't backwards compatible I just disabled the check.